### PR TITLE
Platform: Add initial Samsung Galaxy A52 (a52q) Support

### PR DESCRIPTION
**Pull request by leandrofriedrich**
### Description

This MR adds initial support for the a52q platform from Samsung. 
Currently working are: UFS (in uefi), SimpleInit, Buttons, Screen
Not working: Touch, booting WinPE, UEFI Mass Storage
### Checklist

* [x] Have you tested the change you are submitting?
* [x] Is the commits history nice and clean?

### CODE REVIEW

**Comment by leandrofriedrich** [comment]
@leandrofriedrich (i did not understand how to fill in the checklist)

File: Platform/Samsung/sm7125/a52q.dsc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/sm7125/sm7125.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Samsung/sm7125/a52q.fdf.inc
+
+!include Platform/Qualcomm/sm7125/sm7125.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT -DENABLE_LINUX_SIMPLE_MASS_STORAGE
```
Comment (ungeskriptet) [inline Platform/Samsung/sm7125/a52q.dsc:16]: You can remove `-DENABLE_LINUX_SIMPLE_MASS_STORAGE` and the mainline dtb if mass storage isn't working

Comment (halal-beef) [reply Platform/Samsung/sm7125/a52q.dsc:16]: agreed

**Comment by maj113** [comment]
> (i did not understand how to fill in the checklist)

[ x ]

**Comment by leandrofriedrich** [comment]
> > (i did not understand how to fill in the checklist)
> 
> [ x ]

thx :heart: :hand:

**Comment by maj113** [comment]
> thx ❤️ ✋

oops

**Review by qaz6750 — APPROVED** [review]

**Pull request merged**